src/Components/Image.js

```diff
@@ -11,20 +11,39 @@ type Props = $Component & {
   alt: string,
   width: string,
   title: string,
+  url: string,
 };
 
-const Image = ({ src, height, alt, width, title, classes }: Props) => (
-  <div className={classes.root}>
-    <img
-      src={src}
-      width={width}
-      height={height}
-      alt={alt}
-      title={title}
-      className={classes.image}
-    />
-  </div>
-);
+const Image = ({ src, height, alt, width, title, classes, url }: Props) => {
+  if (url) {
+    return (
+      <div className={classes.root}>
+        <a href={url} title={title} onClick={(e) => e.preventDefault()}>
+          <img
+            src={src}
+            width={width}
+            height={height}
+            alt={alt}
+            title={title}
+            className={classes.image}
+          />
+        </a>
+      </div>
+    );
+  }
+  return (
+    <div className={classes.root}>
+      <img
+        src={src}
+        width={width}
+        height={height}
+        alt={alt}
+        title={title}
+        className={classes.image}
+      />
+    </div>
+  );
+};
```

Comment:
I would make one change to this file to make it easier to maintain changes to it in the future. Note, the below is pseudocode so if you change your code, please re-test locally to make sure I didn't miss anything:

```javascript
const Image = ({ src, height, alt, width, title, classes, url }: Props) => {
  const img = <img
        src={src}
        width={width}
        height={height}
        alt={alt}
        title={title}
        className={classes.image}
      />;

  if (url) {
    return (
      <div className={classes.root}>
        <a href={url} title={title} onClick={(e) => e.preventDefault()}>
          {img}
        </a>
      </div>
    );
  }
  return (
    <div className={classes.root}>
      {img}
    </div>
  );
};

```